tweb/src/main/frontend/src/scss/fundrequest/_alert.scss

```diff
@@ -23,6 +23,16 @@
         &:hover {
             @include box-shadow($z-depth-3);
         }
+
+        &-body {
+            display: inline-block;
+        }
+
+    }
+
+    button.close {
```

Comment:
<img width="430" alt="screen shot 2018-06-18 at 13 25 51" src="https://user-images.githubusercontent.com/1853096/41533780-4bbc0742-72fc-11e8-8d56-1d1645ab131e.png">

When having less room the message goes on 2 line or more, which is good. But the text jumps below the closing icon, and you'll get a bit white space on top.
My suggestion is to try with floating, or position absolute.